lib/commands/exam.js

```diff
@@ -101,6 +117,45 @@ module.exports = TestCommand.extend({
     return this._super.run.apply(this, arguments);
   },
 
+  /**
+   *
+   * @param {Object} option
+   */
+  getExecutionMappigInstance: function(options) {
+    if (executionMapping !== undefined && executionMapping.numberOfBrowsers !== undefined && executionMapping.executionMapping !== undefined) {
+      return executionMapping;
+    }
+    const executionFilePath = options.replayExecution;
+    const browserIdsToReplay = options.replayBrowser;
+
+    let testMapping = {};
+
+    try {
+      // Read the replay execution json file.
+      const executionToReplay = fs.readFileSync(executionFilePath);
+      testMapping = JSON.parse(executionToReplay);
+    } catch (err) {
+      throw new Error(`Error reading reply execution JSON file - ${err}`);
+    }
+
+    const browserModuleMapping = testMapping.executionMapping;
+
+    executionMapping.numberOfBrowsers = testMapping.numberOfBrowsers;
+    executionMapping.executionMapping = ((browserIdsToReplay, browserModuleMapping) => {
```

Comment:
`executionMapping.executionMapping` seems a bit weird to me, since the name of the attribute is the same as the owner/parent.
How about `executionMapping.browserToModuleMap`?